projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-admin-menu.php

```diff
@@ -371,7 +393,7 @@ private function prepare_menu_item_url( $url, $parent_slug = '' ) {
 			)
 		) {
 			if (
-				( 'admin.php' !== $parent_file && file_exists( WP_PLUGIN_DIR . "/$parent_file" ) && ! is_dir( WP_PLUGIN_DIR . "/$parent_file" ) ) ||
+				( false === $admin_is_parent && 'admin.php' !== $parent_file && file_exists( WP_PLUGIN_DIR . "/$parent_file" ) && ! is_dir( WP_PLUGIN_DIR . "/$parent_file" ) ) ||
```

Comment:
Nitpick: I think we can simplify this now to
```suggestion
				( false === $admin_is_parent && file_exists( WP_PLUGIN_DIR . "/$parent_file" ) && ! is_dir( WP_PLUGIN_DIR . "/$parent_file" ) ) ||
```

I wonder if we can even remove the `( file_exists( ABSPATH . "/wp-admin/$parent_file" ) && ! is_dir( ABSPATH . "/wp-admin/$parent_file" )` parts, so the check would be the same as done by Core in https://core.trac.wordpress.org/browser/trunk/src/wp-admin/menu-header.php?rev=49193#L260